### PR TITLE
[BLOG] adding quansight-labs-awarded-three-czi-eoss-cycle5-grants

### PR DESCRIPTION
**Pull request by noatamir**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.
----

Authors to add later: Pamphile Roy, Noa Tamir, Ralf Gommers, Tania Allard

### CODE REVIEW

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+title: ‘Quansight Labs awarded three CZI EOSS Cycle 5 Grants'
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:2]:
```suggestion
title: "Quansight Labs awarded three CZI EOSS Cycle 5 Grants"
```

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+title: "Quansight Labs awarded three CZI EOSS Cycle 5 Grants"
+author: jaime-rodriguez-guerra
+published: November 10, 2022
+description: 'We are delighted to share details about new grants to support the sustainability of SciPy, conda-forge, and CuPy‘
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:5]:
```suggestion
description: "We are delighted to share details about new grants to support the sustainability of SciPy, conda-forge, and CuPy"
```

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+---
+title: "Quansight Labs awarded three CZI EOSS Cycle 5 Grants"
+author: jaime-rodriguez-guerra
+published: November 10, 2022
+description: "We are delighted to share details about new grants to support the sustainability of SciPy, conda-forge, and CuPy"
+category: [Funding, PyData Ecosystem, Packaging, Community]
+featuredImage:
+  src: /posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants/blog_feature_czi5.png
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:8]:
```suggestion
  src: /posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants/blog_feature_czi5.svg
```

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+### Call for contributions
+
+SciPy is open source, and as such, this grant work will also be done in the open and with the broaders community. It means that you can all contribute to help or provide feedback. Head over to https://scipy.org/ and help us shape SciPy for the next decade.
+
+## Transparent, open & sustainable infrastructure for conda-forge and bioconda
+**PIs: [Jaime Rodríguez-Guerra](https://github.com/jaimergp) (Quansight), [Wolf Vollprecht](https://github.com/wolfv) (Quantstack)**
+
+This is a joint project with [QuanstStack](https://quantstack.net/) to improve conda-forge and [Bioconda's](https://bioconda.github.io/) sustainability and transparency by adopting vendor-agnostic and secure infrastructure practices. In addition, the team will develop comprehensive maintenance metrics and dashboards 🎉
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:59]:
```suggestion
This is a joint project with [QuanstStack](https://quantstack.net/) to improve conda-forge and [Bioconda's](https://bioconda.github.io/) sustainability and transparency by adopting vendor-agnostic and secure infrastructure practices. In addition, we will develop comprehensive maintenance metrics and dashboards. 🎉
```

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+* Unit tests added to cover all possible input combinations and edge cases, e.g. overflows, exceptions, and warnings
+
+### Maintain CuPy to continue supporting the latest platforms
+As with other GPU applications, CuPy heavily relies on NVIDIA CUDA and AMD ROCm libraries. Currently, CuPy supports with CUDA versions (10.2, 11.0-11.6) and four ROCm versions (4.0, 4.2, 4.3, 5.0). GPU vendors generally release updates of these toolkits on a quarterly basis, and providing immediate support for them is essential to allow users to enjoy the latest features, performance enhancements, security, and bug fixes. This work includes:
+
+* Implementing Python wrappers for new APIs added to the library to make them available to CuPy
+* Supporting  new features from a new CUDA or ROCm release, when applicable 
+* Setting up a continuous integration system to provide extensive coverage of the supported software combinations
+* Building a delivery pipeline to provide binary packages for the supported versions of the library
+
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:103]:
```suggestion


```

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+Today, we are thrilled to announce that three new EOSS Cycle 5 grants were awarded, in which [Quansight Labs team members are taking a leading role](link to announcement).
+
+## SciPy: Fundamental Tools for Biomedical Research
+**PIs: [Matt Haberland](https://github.com/mdhaber) (Cal Poly), [Pamphile Roy](https://github.com/tupui) (Quansight)**
+
+We partnered with [Cal Poly](https://www.calpolycorporation.org/) to improve the scientific Python library SciPy towards helping the Biomedical community.
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:24]:
```suggestion
We partnered with [Cal Poly](https://www.calpolycorporation.org/) to improve the scientific Python library [SciPy](https://scipy.org/) towards helping the Biomedical community.
```

Comment (noatamir) [reply apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:24]: add link to project

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+### Call for contributions
+
+SciPy is open source, and as such, this grant work will also be done in the open and with the broaders community. It means that you can all contribute to help or provide feedback. Head over to https://scipy.org/ and help us shape SciPy for the next decade.
+
+## Transparent, open & sustainable infrastructure for conda-forge and bioconda
+**PIs: [Jaime Rodríguez-Guerra](https://github.com/jaimergp) (Quansight), [Wolf Vollprecht](https://github.com/wolfv) (Quantstack)**
+
+This is a joint project with [QuanstStack](https://quantstack.net/) to improve conda-forge and [Bioconda's](https://bioconda.github.io/) sustainability and transparency by adopting vendor-agnostic and secure infrastructure practices. In addition, the team will develop comprehensive maintenance metrics and dashboards 🎉
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:59]:
```suggestion
This is a joint project with [QuanstStack](https://quantstack.net/) to improve [conda-forge](https://conda-forge.org/) and [Bioconda's](https://bioconda.github.io/) sustainability and transparency by adopting vendor-agnostic and secure infrastructure practices. In addition, the team will develop comprehensive maintenance metrics and dashboards 🎉
```

Comment (noatamir) [reply apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:59]: add link to project

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+**PI: [Kenichi Maehashi](https://github.com/kmaehashi) (Preferred Networks)**
+
+Led by [Preferred Networks](https://www.preferred.jp/en), Quansight Labs is proud to collaborate on this grant, which aims to provide a series of GPU accelerated routines for signal processing and interpolation in CuPy. The goal is to match SciPy’s `signal` and `interpolate` submodules,  as a foundation for the research community. 
+
+This project has two main goals
+1. Develop interpolation and signal processing modules
+2. Maintain CuPy to continue supporting the latest platforms
+
+### Develop interpolation and signal processing modules
+The CuPy project aims to reach full [coverage of the NumPy and SciPy APIs](https://docs.cupy.dev/en/latest/reference/comparison.html). The developers will implement, profile, document, and add unit tests for each API in the targeted modules: `cupyx.scipy.interpolate` and `cupyx.scipy.signal`. This work includes:
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:89]:
```suggestion
The [CuPy project](https://cupy.dev/) aims to reach full [coverage of the NumPy and SciPy APIs](https://docs.cupy.dev/en/latest/reference/comparison.html). The developers will implement, profile, document, and add unit tests for each API in the targeted modules: `cupyx.scipy.interpolate` and `cupyx.scipy.signal`. This work includes:
```

Comment (noatamir) [reply apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:89]: add link to project

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+Although SciPy is well established in the scientific Python ecosystem, its broad scope makes it difficult for a user to notice all its functionalities. Examples of libraries duplicating some “hidden” features of SciPy are numerous. This is a general observation in Science, where things are sometimes forgotten for years, if not centuries. The Fast Fourier Transform was discovered by Gauss more than a century before Fourier, Cooley and Tukey re-discovered it.
+
+The dissemination of scientific knowledge and tools is vital to the open source mission and sustainability of projects like SciPy. To achieve this, the team is making time for ongoing outreach activities such as community calls, and is active on community platforms like Slack and Discord. In addition, the team is planning to promote SciPy’s features at conferences and write specific tutorials targeting Biomedical researchers.
+
+### Call for contributions
+
+SciPy is open source, and as such, this grant work will also be done in the open and with the broaders community. It means that you can all contribute to help or provide feedback. Head over to https://scipy.org/ and help us shape SciPy for the next decade.
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:54]:
```suggestion
SciPy is open source, and as such, this grant work will also be done in the open and with the broaders community. It means that you can all contribute to help or provide feedback. Join [our community](https://scipy.org/community/), and help us shape SciPy for the next decade.
```

Comment (noatamir) [reply apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:54]: avoid repetition from previous feedback section where "head over" is already used.

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+### Maintain CuPy to continue supporting the latest platforms
+As with other GPU applications, CuPy heavily relies on NVIDIA CUDA and AMD ROCm libraries. Currently, CuPy supports with CUDA versions (10.2, 11.0-11.6) and four ROCm versions (4.0, 4.2, 4.3, 5.0). GPU vendors generally release updates of these toolkits on a quarterly basis, and providing immediate support for them is essential to allow users to enjoy the latest features, performance enhancements, security, and bug fixes. This work includes:
+
+* Implementing Python wrappers for new APIs added to the library to make them available to CuPy
+* Supporting  new features from a new CUDA or ROCm release, when applicable 
+* Setting up a continuous integration system to provide extensive coverage of the supported software combinations
+* Building a delivery pipeline to provide binary packages for the supported versions of the library
+
+
+# Want to join?
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:105]: @gabalafou the spacing here seems a bit short to me. Is this a known issue with a header after a list perhaps?
Screenshot of the heading Want to Join?

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+The projects mentioned in this blog post are open-source community-led projects across the PyData ecosystem. Thanks to this round of CZI funding, Quansight Labs will contribute more to SciPy, conda-forge and CuPy., We also welcome and encourage your help in this endeavor. If you have a particular interest in one of these projects, or are interested in the work we do at Quansight and Quansight Labs, have a look at our [open job postings](https://www.quansight.com/careers). We are a fully remote and globally distributed company. Please don't hesitate to contact us with questions regarding your application.
```
Comment (tupui) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:107]:
```suggestion
The projects mentioned in this blog post are open-source community-led projects across the PyData ecosystem. Thanks to this round of CZI funding, Quansight Labs will contribute more to SciPy, conda-forge and CuPy. We also welcome and encourage your help in this endeavor. If you have a particular interest in one of these projects, or are interested in the work we do at Quansight and Quansight Labs, have a look at our [open job postings](https://www.quansight.com/careers). We are a fully remote and globally distributed company. Please don't hesitate to contact us with questions regarding your application.
```

**Review by tupui — APPROVED** [review]
SciPy part LGTM, thanks!

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
+---
+title: "Quansight Labs awarded three CZI EOSS Cycle 5 Grants"
+author: jaime-rodriguez-guerra
```
Comment (tupui) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:3]: Shouldn't all recipient be listed?

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+published: November 10, 2022
+description: "We are delighted to share details about new grants to support the sustainability of SciPy, conda-forge, and CuPy"
+category: [Funding, PyData Ecosystem, Packaging, Community]
+featuredImage:
+  src: /posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants/blog_feature_czi5.svg
+  alt: ‘Logos of CZI, conda-forge, scipy, and CuPy'
+hero:
+  imageSrc: /posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants/blog_hero_czi5.svg
```
Comment (tupui) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:11]: Somehow SciPy's logo is cut which is not good. And Conda-Forge is blurry (does not seem to be happening for the featuredImage on the blog posts list)

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
+---
+title: "Quansight Labs awarded three CZI EOSS Cycle 5 Grants"
+author: jaime-rodriguez-guerra
+published: November 10, 2022
+description: "We are delighted to share details about new grants to support the sustainability of SciPy, conda-forge, and CuPy"
+category: [Funding, PyData Ecosystem, Packaging, Community]
+featuredImage:
+  src: /posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants/blog_feature_czi5.svg
+  alt: ‘Logos of CZI, conda-forge, scipy, and CuPy'
```
Comment (tupui) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:9]:
```suggestion
  alt: ‘Logos of CZI, conda-forge, SciPy, and CuPy'
```

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+hero:
+  imageSrc: /posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants/blog_hero_czi5.svg
+  imageAlt: 'Logos of CZI, conda-forge, scipy, and CuPy'
```
Comment (tupui) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:12]:
```suggestion
  imageAlt: 'Logos of CZI, conda-forge, SciPy, and CuPy'
```

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md (hunk 12 above)
Comment (noatamir) [reply apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:3]: It's not possible at the moment. There's an open tracking issue to add multiple authors once the blog system is updated to resolve this issue. I've added this post there.

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md (hunk 13 above)
Comment (noatamir) [reply apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:11]: I don't have a higher res version of the conda-forge logo. Maybe @jaimergp can help with that? This is what I could get from their repository and website.
The SciPy logo is only cut in the preview due to a known bug #555. It should be ok in the live version.

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+
+### Call for contributions
+
+SciPy is open source, and as such, this grant work will also be done in the open and with the broaders community. It means that you can all contribute to help or provide feedback. Join [our community](https://scipy.org/community/), and help us shape SciPy for the next decade.
+
+## Transparent, open & sustainable infrastructure for conda-forge and bioconda
+**PIs: [Jaime Rodríguez-Guerra](https://github.com/jaimergp) (Quansight), [Wolf Vollprecht](https://github.com/wolfv) (Quantstack)**
+
+This is a joint project with [QuanstStack](https://quantstack.net/) to improve conda-forge and [Bioconda's](https://bioconda.github.io/) sustainability and transparency by adopting vendor-agnostic and secure infrastructure practices. In addition, we will develop comprehensive maintenance metrics and dashboards. 🎉
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:59]:
```suggestion
This is a joint project with [QuanstStack](https://quantstack.net/) to improve [conda-forge](https://conda-forge.org/) and [Bioconda's](https://bioconda.github.io/) sustainability and transparency by adopting vendor-agnostic and secure infrastructure practices. In addition, we will develop comprehensive maintenance metrics and dashboards. 🎉
```

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md (hunk 10 above)
Comment (gabalafou) [reply apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:105]: For now we can insert a `<br />` to put more space between the list and the heading.

```suggestion
<br />

# Want to join?
```

Comment (bskinn) [reply apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:105]: I'm pretty sure this issue isn't relevant to the Consulting site, at least up to the present. The Storyblok content editor has given us a fair measure of control over vertical spacing like this. (It's sometimes a bit awkward to make it produce the vertical spacing we want, but it's usually possible.)

Once #396 is completed, though, then this _will_ be relevant to the Consulting blog also. 

---

I'm a bit hesitant to declare this a uniform styling problem with bulleted lists.  How is the spacing for a bulleted list followed by a normal paragraph? If it looks ok, then this is likely just an issue for when a bulleted list ends a section. If so, I think we'd want any style fix to be narrowly scoped, or else the spacing might be too *big* in some cases---and that would be harder to patch in Markdown than too little space.
